tests/test_model_specs.py

```diff
@@ -57,35 +57,184 @@ def test_model_specs_are_valid(self):
             # Validate top-level keys are correct
             with self.subTest(metadata.pyname):
                 self.assertTrue(
-                    required_keys.issubset(model.ARGS_SPEC),
-                    ("Required key(s) missing from ARGS_SPEC: "
-                     f"{set(required_keys).difference(model.ARGS_SPEC)}"))
-                extra_keys = set(model.ARGS_SPEC).difference(required_keys)
+                    required_keys.issubset(model.MODEL_SPEC),
+                    ("Required key(s) missing from MODEL_SPEC: "
+                     f"{set(required_keys).difference(model.MODEL_SPEC)}"))
+                extra_keys = set(model.MODEL_SPEC).difference(required_keys)
                 if (extra_keys):
                     self.assertEqual(extra_keys, set([optional_spatial_key]))
                     self.assertTrue(
-                        set(model.ARGS_SPEC[optional_spatial_key]).issubset(
+                        set(model.MODEL_SPEC[optional_spatial_key]).issubset(
                             {'spatial_keys', 'different_projections_ok'}))
 
             # validate that each arg meets the expected pattern
             # save up errors to report at the end
-            for key, arg in model.ARGS_SPEC['args'].items():
+            for key, arg in model.MODEL_SPEC['args'].items():
                 # the top level should have 'name' and 'about' attrs
                 # but they aren't required at nested levels
-                self.validate(arg, f'{model_name}.{key}')
+                self.validate(arg, f'{model_name}.args.{key}')
```

Comment:
Should we update this function to `validate_args`?

Reply:
done!